deepchem/models/torch_models/molgan.py

```diff
@@ -0,0 +1,82 @@
+from typing import  List, Tuple #, Any, Optional,
+
+# import torch
+# from torch import nn
+# from torch.nn import functional as F
+# from deepchem.feat.molecule_featurizers.molgan_featurizer import GraphMatrix
+from deepchem.models import WGAN
+# from deepchem.models.layers import MolGANEncoderLayer
+
+
+class BasicMolGANModel(WGAN):
```

Comment:
Make sure to add this new class to the docs

Reply:
Have added this model to the documentation